tests/torch/modules/test_rnn.py

```diff
@@ -392,7 +394,9 @@ def test_export_stacked_bi_lstm(tmp_path):
     model, algo = create_compressed_model_and_algo_for_test(test_rnn, config)
 
     test_path = str(tmp_path.joinpath('test.onnx'))
-    algo.export_model(test_path)
+    # Exporting the operator ::chunk to ONNX opset version 9 is not supported.
+    # Support for this operator was added in version 11
+    algo.export_model(test_path, save_format='onnx_11')
```

Comment:
By default, opset 10 is used and export fails with `Unsupported: ONNX export of operator prim::ConstantChunk, unknown dimension size.`